src/PerfCounters_x86.h

```diff
@@ -85,6 +85,7 @@ static CpuMicroarch compute_cpu_microarch() {
     case 0x906a0:
       return IntelAlderlake;
     case 0xb0670:
+    case 0xb06a0: // i9-13900H (Raptor Lake-H) / i7-1355U (Raptor Lake-U)
```

Comment:
Goes for my `i7-13700H` too, so maybe skip the comment like for the rest of them (or add this too)?

Reply:
Yeah we don't need this comment.

Reply:
we do have it below... I'll drop the specific cpus

Reply:
So @jtunhag What is your test result with the updated rev (or this change applied to master yourself)?

Reply:
We have them for AMD because AMD has that weird logic for `ext_family`. Intel is pretty trivial by comparison.